Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyStringComboBoxCtrl.cpp

```diff
@@ -46,7 +46,7 @@ namespace AzToolsFramework
 
     void PropertyStringComboBoxCtrl::Add(const AZStd::vector<AZStd::string>& vals)
     {
-        GetComboBox()->clear();
+        ComboBoxBase::clearElements();
```

Comment:
Why do you need to qualify this call with `ComboBoxBase::`? I'd expect this to be just
```suggestion
        clearElements();
```